### DISCUSSION TITLE
Please provide ZIP Archive to download at Github

### DISCUSSION THREAD

**Discussion by lunainvictum** [File-New-Project/EarTrumpet#751] [Ideas]
Happy new year together!

Could it be possible to provide a simple ZIP Archive to download here at Github?

I dont have MS Store. I removed the complete "App Support" from my Windows so it isnt possible to use and install any "Apps" anymore. Like i want it.

And i dont want to use Chocolatey either. 

So i look at the Page from Chocolatey and searched the package. There i looked in the Script where chocolatey download eartrumpet and found out that it simply provides a Release.zip. So i download the nupkg. Decompressed it with 7zip. Searched for Release.zip, decompressed this too and found eartrumpet. 

This is to complicated for normal peoples, who wants to use eartrumpet, without to sell their Soul to Microsoft or Chocolatey. 

Sry for my bad english. Im native German Speaker.

**Comment by riverar** [comment]
How do you plan on receiving updates?

**Reply by zachary-wang12** [reply]
Actually, if all functions work well, there is no need to update. At the same time, update may be checked via github release page automatically or manually.

**Comment by lunainvictum** [comment]
So long everything works like needed, no Updates are necessary.

And if a Problem occurs, i could simply Check here at github if a new Release is released.

To write a Option or Function into eartrumpet to Check at github if a new Version is released and update it by simply download it from github and decompressing the archive on the installation path is possible too.

I wrote a powershell Script for a another program that i found at github and dont have a Update function (only to Check if a new Release is released and to open the webpage for the User). 

Am 2. Januar 2021 01:45:21 MEZ schrieb Rafael Rivera <notifications@github.com>:
>How do you plan on receiving updates?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/File-New-Project/EarTrumpet/issues/606#issuecomment-753410788

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by Trouffman** [comment]
The main reason for the release would be to not be tied to the Windows Store.

**Comment by Trouffman** [comment]
Actually there is a way (still via the Store),

Use Winget : winget install -e File-New-Project.EarTrumpet

Problem Solved. No need anything else. :)

**Comment by IcePlains** [comment]
Thank you very much for this solution, @lunainvictum. I could not find elegant way to install it myself.

I wanted fully offline solution, and this is it. This does not have updating, but i dont need that.

So in short:
1. Download from https://community.chocolatey.org/packages/eartrumpet
2. Open .nupkg with 7z or similar
3. Extract \tools\release.zip, unpack into some folder in your profile, for instance c:\Users\user\AppData\Local\EarTrumpet\
4. Make shortcut to EarTrumpet.exe and place it in startup folder: c:\Users\user\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\

**Reply by riverar** [reply]
May I ask why you think you don't need updating?

**Reply by IcePlains** [reply]
Updates are appreciated, and important. But for applications that do not need internet access as their function, offline installer is the better approach imo.

Additionally, if computer has Microsoft Store disabled and/or cut from the internet, current installer presents additional challenges.
And i currently do not have any package managers on my computer, so installing package manager just for EarTrumpet means additional software that is not really needed.

EarTrumpet has been stable in the past, so manually checking for updates once in a while is good enough for me.

Thank you for awesome piece of software.

**Reply by ghost** [reply]
> May I ask why you think you don't need updating?

People update their stuff if they want so. I update my programs myself, without the use of any package manager. The fact that there isn't a regular release on github is but an inconvenience.
